packages/@aws-cdk-testing/framework-integ/package.json

```diff
@@ -46,7 +46,9 @@
     "aws-cdk-lib": "0.0.0",
     "cdk8s": "2.69.10",
     "cdk8s-plus-27": "2.9.5",
-    "constructs": "^10.0.0"
+    "constructs": "^10.0.0",
+    "jsii": "5.7.1",
+    "jsii-reflect": "1.105.0"
```

Comment:
could you explain why do you add these dependencies?

Reply:
As mentioned in the PR message, using the `jsii` packages in the integration allows me to differentiate `@deprecated` enum values, and to test accordingly:

https://github.com/aws/aws-cdk/blob/1e8b3766f3ddf38120a41d39776aadb610e10a20/packages/%40aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.windows-machine-image.ts#L60-70